lib/controller/EetplanController.class.php

```diff
@@ -39,13 +43,41 @@ public function view() {
 	}
 
 	public function noviet($uid = null) {
+	    $eetplan = $this->model->getEetplanVoorNoviet($uid);
+        if ($eetplan === false) {

```

Comment:
Waarom heb je nog access control in deze functie? Kan dat niet in de contructor ACL of in de performAction() hierboven?